resources/resources_dev.go

```diff
@@ -0,0 +1,9 @@
+//go:build !release
+// +build !release
+
+package resources
```

Comment:
We may have to move to a different package? As long as it's in the resource package, it will be packaged into our bytebase binary. However, mysql is only used for testing if I understand correctly.